client/src/components/auth/Login.js

```diff
@@ -21,15 +21,23 @@ export default class Login extends Component {
   async handleSubmit(event) {
     event.preventDefault()
 
-    let res
     try {
-      res = await this.props.login()
-      if (res.error) {
-        this.setState({ ...this.state, error: res.error })
-        window.alert('error')
-      } else {
-        this.setState({ ...this.state, error: '' })
-      }
+      
+      const requestOptions = {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json', 'Access-Control-Allow-Origin': "*" },
+        body: JSON.stringify({ email: this.state.email, password: this.state.password })
+      };
+
+      fetch('/users/login', requestOptions)
+          .then(res => {
+            if (res.status === 200) {
+                this.props.login();
+            } else {
+              console.log(res.message);
+            }
+          })
+        
     } catch (error) {
```

Comment:
this catch would need to be chained at the end of line 39 as `.catch(err => ...)`